packages/devextreme/js/localization/messages/en.json

```
@@ -274,7 +274,7 @@
         "dxScheduler-appointmentAriaLabel-recurring": "Recurring appointment",
 
         "dxScheduler-appointmentListAriaLabel": "Appointment list",
-
+        "dxScheduler-newAppointmentPopupTitle": "New Appointment",
```

To keep the new variable naming consistent with the existing "editPopupTitle", I propose the following:
```suggestion
        "dxScheduler-newPopupTitle": "New Appointment",
```

---

Applied this change.